docs/products/kafka/kafka-connect/howto/debezium-source-connector-sql-server.rst

```diff
@@ -0,0 +1,193 @@
+Create a Debezium source connector for SQL Server
+==================================================
+
```

Comment:
Strictly speaking this is where you introduce CDC, so you can even add "(CDC)" here and omit it later.
But your current approach also works for me.